centaur/src/main/resources/standardTestCases/papi_cpu_platform/papi_cpu_platform.wdl

```diff
@@ -0,0 +1,27 @@
+version 1.0
+
+task cpu_platform {
+    input {
+        String cpu_platform
+    }
+    command {
+       apt-get install --assume-yes jq > /dev/null
+       NAME=`curl -s -H "Metadata-Flavor: Google" http://metadata.google.internal/computeMetadata/v1/instance/name`
```

Comment:
I'm pretty sure `shellcheck` prefers (and tells me to prefer) the `$(...)` style to the ``` `...` ``` style of "run a command, record the result"